root/etc/cont-init.d/98-julia

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/with-contenv bash
+
+gh_lates() {
+  curl -s https://api.github.com/repos/$1/$2/releases/latest \
+    | grep "tag_name" \
+    | cut -d : -f 2,3 \
+    | tr -d '\"\ v,'
+}
+
+download_vsix() {
+  wget -qO $1.$2-$3.vsix https://$1.gallery.vsassets.io/_apis/public/gallery/publisher/$1/extension/$2/$3/assetbyname/Microsoft.VisualStudio.Services.VSIXPackage
+}
```

Comment:
Please use the helper function to install extentions. https://github.com/linuxserver/docker-mods/tree/template#tips-and-tricks

Reply:
Is there documentation on how to use this tool?

Reply:
We have a example here https://github.com/linuxserver/docker-mods/blob/code-server-terraform/root/etc/cont-init.d/98-terraform#L8

Reply:
I am now using the helper instead of `wget`ing the .vsix files and installing them. In your example there was no version specified so I assume it just grabs the latest. It should also mean that the code is now `openvscode`-compatible.